target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -159,17 +191,19 @@ TARGET_DEVICES += solidrun_clearfog-base-a1
 
 define Device/globalscale_mirabox
   $(Device/NAND-512K)
+  DEVICE_VENDOR := Globalscale
+  DEVICE_MODEL := Mirabox
   DEVICE_DTS := armada-370-mirabox
   SUPPORTED_DEVICES += mirabox
-  DEVICE_TITLE := Globalscale Mirabox
 endef
 TARGET_DEVICES += globalscale_mirabox
 
 define Device/cznic_turris-omnia
+  DEVICE_VENDOR := Turris
+  DEVICE_MODEL := Omnia
```

Comment:
Hmm, if I look at the definition name, it could as well be:
DEVICE_VENDOR := CZ.NIC
DEVICE_MODEL := Turris Omnia

Reply:
ack